framework/core/js/src/admin/components/CreateUserModal.tsx

```diff
@@ -0,0 +1,241 @@
+      </>
+    );
+  }
+
+  body() {
+    return (
+      <>
+        <div className="Form Form--centered">{this.fields().toArray()}</div>
+      </>
+    );
+  }
+
+  fields() {
+    const items = new ItemList();
+
+    const usernameLabel = extractText(app.translator.trans('core.admin.create_user.username_placeholder'));
+    const emailLabel = extractText(app.translator.trans('core.admin.create_user.email_placeholder'));
+    const emailConfirmationLabel = extractText(app.translator.trans('core.admin.create_user.email_confirmed_label'));
+    const useRandomPasswordLabel = extractText(app.translator.trans('core.admin.create_user.use_random_password'));
+    const passwordLabel = extractText(app.translator.trans('core.admin.create_user.password_placeholder'));
+
+    items.add(
+      'username',
+      <div className="Form-group">
+        <input
+          className="FormControl"
+          name="username"
+          type="text"
+          placeholder={usernameLabel}
+          aria-label={usernameLabel}
+          bidi={this.username}
+          disabled={this.loading}
+        />
+      </div>,
+      100
+    );
+
+    items.add(
+      'email',
+      <div className="Form-group">
+        <input
+          className="FormControl"
+          name="email"
+          type="email"
+          placeholder={emailLabel}
+          aria-label={emailLabel}
+          bidi={this.email}
+          disabled={this.loading}
+        />
+      </div>,
+      80
+    );
+
+    items.add(
+      'emailConfirmation',
+      <div className="Form-group">
+        <Switch
+          name="emailConfirmed"
+          state={this.requireEmailConfirmation()}
+          onchange={(checked: boolean) => this.requireEmailConfirmation(checked)}
+          disabled={this.loading}
+        >
+          {emailConfirmationLabel}
+        </Switch>
+      </div>,
+      60
+    );
+
+    items.add(
+      'password',
+      <div className="Form-group">
+        <Switch
+          name="useRandomPassword"
+          state={this.password() === null}
+          onchange={(enabled: boolean) => {
+            this.password(enabled ? null : '');
+          }}
+          disabled={this.loading}
+        >
+          {useRandomPasswordLabel}
+        </Switch>
+
+        <input
+          className="FormControl"
+          name="password"
+          type="password"
+          autocomplete="new-password"
+          placeholder={passwordLabel}
+          aria-label={passwordLabel}
+          bidi={this.password}
+          disabled={this.loading || this.password() === null}
+        />
+      </div>,
+      40
+    );
+
+    items.add(
+      'bulkAdd',
+      <div className="Form-group CreateUserModal-bulkAdd">
+        <Switch name="bulkAdd" disabled={this.loading} state={this.bulkAdd()} onchange={(checked: boolean) => this.bulkAdd(checked)}>
+          {app.translator.trans('core.admin.create_user.bulk_add_label')}
+        </Switch>
+      </div>,
+      0
+    );
```

Comment:
I have been using laravel filaments lately and quite like how they use a `Create & create another button` (https://demo.filamentphp.com/shop/products/create) maybe we might consider a similar approach here? wdyt